desktop/src/main/java/bisq/desktop/main/overlays/windows/OfferDetailsWindow.java

```diff
@@ -253,21 +253,20 @@ private void addContent() {
             addConfirmationLabelLabel(gridPane, ++rowIndex, Res.get("offerDetailsWindow.myTradingAccount"), myPaymentAccount.getAccountName());
         } else {
             final String method = Res.get(paymentMethod.getId());
-            String methodWithBankId = method + bankId;
             String paymentMethodLabel = Res.get("shared.paymentMethod");
             if (countryCode != null && (isNationalBanks || isSpecificBanks || isSepa)) {
```

Comment:
For specific banks payment method we do want to show the bank name and/or id as it would be otherwise for someone to match ones offer using this payment method.